### DISCUSSION TITLE
Jump from Github Pages to Github

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22113] [Other Feature Feedback, Questions, & Ideas]
<p>I was wondering if there was a way to go to the current file in Github.</p>
<p>For example, If I’m at a <code>foo.md</code> file, I’d like to go to the location of <code>foo.md</code> on github. Is such a thing possible with GH Pages?</p>

**Comment by techboyg5** [comment, marked as answer]
<p>No, You have to add links manually to every page.</p>
